### ISSUE TITLE
Check if window is already fullscreen or borderless

### ISSUE THREAD

**Issue by idietmoran** [idietmoran/Display-Lock#68]
There's currently a bug where if the window is already fullscreen or borderless when the user hit's start, the application with add a border to the window. This behavior should be avoided.

**Comment by Arbalest007** [comment]
@idietmoran If possible I would like to take a look at this issue as well.

**Comment by idietmoran** [comment]
Sure, check win.c in dispLib/

**Comment by Arbalest007** [comment]
@idietmoran Just wanted to clarify regarding this issue. When I have a window in fullscreen mode with DisplayLock activated, and I press the "Start" button (Windows Key), I get the Start Menu popping up, and I am able to move the cursor out of the window. Are you trying to disable the ability to move out of the window once the "Start" button is pressed?

**Comment by idietmoran** [comment]
No, if the window is already fullscreen or already borderless, I want to make sure we don't change the window's styles.
